src/main/java/io/cdap/plugin/servicenow/source/ServiceNowRecordReader.java

```diff
@@ -118,7 +118,7 @@ private void fetchSchema(ServiceNowTableAPIClientImpl restApi) {
       }
 
       schema = Schema.recordOf(tableName, schemaFields);
-    } catch (OAuthProblemException | OAuthSystemException e) {
```

Comment:
Need to consistent like other places. If these are the only ones then fine, but the same should apply for ValidateTable as well.

Reply:
`validateTable` function takes `FailureCollector` as an argument and is expected to not fail and any exception that occurs during function execution should be added to the failure collector.
On the other hand, `RecordReader` functions can fail for any reason and the caller is expected to handle it.